draft-ietf-jsonpath-base.md

```diff
@@ -231,8 +232,7 @@ Children (of a node):
 : If the node is an array, the nodes of its elements.
   If the node is an object, the nodes of its member values.
   If the node is neither an array nor an object, it has no children.
-  Note that the members and member names of an object do not
-  have nodes.
+  Note that members and member names are not children of an object.
```

Comment:
Perhaps we should sharpen the definitions instead:

```
Member:  A name/value pair in an object.  (Neither itself a value, nor a node.)
```

```
Name:  The name in a name/value pair constituting a member.  (Not itself a node.)
```

Reply:
We had (something like) that previously. I prefer to avoid the forward references to "node" because some readers may carry all sorts of assumptions about the term "node" before they read the actual definition.